Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -6086,6 +6089,7 @@
 			isa = PBXGroup;
 			children = (
 				6560A4CF04B3B3E7008AE952 /* CoreFoundation.framework */,
+				7BBCEE65294C850200AF8C8D /* Foundation.framework */,
 				51F0EB6105C86C6B00E6DF1B /* Foundation.framework */,
```

Comment:
Why 2 instances of the same framework?

Reply:
> Why 2 instances of the same framework?

It appears I added the framework in invalid way, it took it from the SDKROOT. Fixed, thanks.